distribution/docs/src/main/resources/_contents/_core-concepts/ingest-intro-contents.adoc

```diff
@@ -1,6 +1,9 @@
 
-placeholder text.
-////
-TODO: write this section.
-See https://codice.atlassian.net/browse/DDF-2471
-////
+Ingest is the process of bringing data products, metadata, or both into the catalog to enable search, sharing, and discovery.
+Ingested files are "transformed" into a neutral format that can be searched against as well as migrated to other formats and systems.
```

Comment:
Is there a reason we put `transformed` in double quotes? Not sure if you wanted to link to that section here if a reader wanted more details on it?